compiler/rustc_codegen_cranelift/scripts/cargo.sh

```diff
@@ -10,7 +10,7 @@ cmd=$1
 shift || true
 
 if [[ "$cmd" = "jit" ]]; then
-cargo +${TOOLCHAIN} rustc $@ -- --jit
+cargo +${TOOLCHAIN} rustc "$@" -- --jit
 else
-cargo +${TOOLCHAIN} $cmd $@
+cargo +${TOOLCHAIN} $cmd "$@"
```

Comment:
These have already been fixed https://github.com/bjorn3/rustc_codegen_cranelift/pull/1109, which hasn't been pulled into rust-lang/rust yet. This is going to conflict when I pull it.

Reply:
Pulled as https://github.com/bjorn3/rustc_codegen_cranelift/commit/11dd005a88eeb6129c8c19ab02cba608d3b7d682.